### PR TITLE
Add llama2-13b-chat-v1 support to `chat_models.BedrockChat`

### PR DESCRIPTION
**Pull request by WouterDurnez**
Hi 👋 We are working with Llama2 on Bedrock, and would like to add it to Langchain. We saw a [pull request](https://github.com/langchain-ai/langchain/pull/13322) to add it to the `llm.Bedrock` class, but since it concerns a chat model, we would like to add it to `BedrockChat` as well.

  - **Description:** Add support for Llama2 to `BedrockChat` in `chat_models`  
  - **Issue:** the issue # it fixes (if applicable) [#13316](https://github.com/langchain-ai/langchain/issues/13316)
  - **Dependencies:** any dependencies required for this change `None`
  - **Tag maintainer:** /
  - **Twitter handle:** `@SimonBockaert @WouterDurnez`

### CODE REVIEW

**Review by 3coins — APPROVED** [review]
@WouterDurnez 
Thanks for making this change. LGTM!

**Comment by baskaryan** [comment]
thanks @WouterDurnez!

File: libs/langchain/langchain/chat_models/bedrock.py [hunk 1]
```diff
@@ -26,6 +27,8 @@ def convert_messages_to_prompt(
     ) -> str:
         if provider == "anthropic":
             prompt = convert_messages_to_prompt_anthropic(messages=messages)
```
Comment (pressoholics) [inline libs/langchain/langchain/chat_models/bedrock.py:29]: @baskaryan @WouterDurnez If using a bedrock anthropic model this always resolves to NotImplementedError due to incorrect implementation of If statement. This should be elseif or immediately return the prompt value once match with anthropic. This is currently an breaking change.

Comment (WouterDurnez) [reply libs/langchain/langchain/chat_models/bedrock.py:29]: 🙈 https://github.com/langchain-ai/langchain/pull/13684 --> added a fix

Comment (WouterDurnez) [reply libs/langchain/langchain/chat_models/bedrock.py:29]: 🙈 Sorry, missed that. Added a fix but then saw the PR above. Thx!

**Pull request merged**